src/com/mendix/widget/ProgressBar/components/Alert.ts

```diff
@@ -0,0 +1,6 @@
+import { DOM, StatelessComponent } from "react";
+
+export const Alert: StatelessComponent<{ message?: string }> = (props) =>
+    props.message
+        ? DOM.div({ className: "alert alert-danger widget-progressbar-alert" }, props.message)
+        : null as any;
```

Comment:
Remove `as any` update typing to
"@types/react": "^15.0.6",
"@types/react-dom": "^0.14.22",